File: 프로그래머스/lv1/12950. 행렬의 덧셈/행렬의 덧셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int lengthOne = arr1.length;
        int lengthTwo = arr1[0].length;
        int[][] answer = new int[lengthOne][lengthTwo];

        for (int i = 0; i < lengthOne; i++) {
            for (int j = 0; j < lengthTwo; j++) {
                answer[i][j] = arr1[i][j] + arr2[i][j];
            }
        }

        return answer;
    }
}

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.js
function solution(arr1, arr2) {
    for(let i = 0; i < arr1.length; i++) {
        for(let j = 0; j < arr1[0].length; j++) {
            arr1[i][j] = arr1[i][j] + arr2[i][j];
        }
    }
    
    return arr1;
}